src/Tickets/Seating/Admin/Tabs/Maps.php

```diff
@@ -50,13 +51,15 @@ public static function get_id(): string {
 	 */
 	public function render(): void {
 		$context = [
-			'cards'       => [],
-			'add_new_url' => add_query_arg( [
-				'page' => Admin::get_menu_slug(),
-				'tab'  => Map_Edit::get_id()
-			] ),
+			'cards'       => Maps_Service::fetch_all(),
```

Comment:
Here you're extracting objects, change this to extract `Map_Card[]`.

Reply:
The method name should indicate the nature of the transformation, e.g. `get_in_card_format`.